package.json

```diff
@@ -17,7 +17,7 @@
     "test:unit": "cross-env BABEL_ENV=test jest --maxWorkers=3",
     "test:integration": "jest --config=test/jest.config.js --maxWorkers=3",
     "test:smoke": "config/scripts/smoke.sh",
-    "test:ts": "tsc -p test/typings/tsconfig.json",
+    "test:ts": "tsc -p test-d.tsconfig.json",
```

Comment:
Moving this config to the root so that we can write `*.test-d.ts` files next to the source code. We still keep the `test/typings` for the feature-based typings that can't be placed near a single module.